backend/src/index.ts

```diff
@@ -296,77 +306,98 @@ app.get("/user", async (req, res: Response) => {
   });
 });
 
-app.post("/society/create", async (req:TypedRequest<CreateSocietyBody>, res: Response) => {
-  const society = req.body;
-  if (!society.name || !society.userId) {
-    return res.status(400).json({ message: 'Invalid input.' });
-  }
-
-  if (!society.profilePicture) {
-    society.profilePicture = "null";
-  }
+app.post(
+  "/society/create",
+  async (req: TypedRequest<CreateSocietyBody>, res: Response) => {
+    const society = req.body;
+    if (!society.name) {
+      return res.status(400).json({ message: "Invalid input." });
+    }
 
-  const sessionFromDB = await validateSession(req.session ? req.session : null);
-  if (!sessionFromDB) {
-    return res.status(401).json({ message: 'Invalid session provided.' });
-  }
-  
-  const newSociety = await prisma.society.create({
-    data: {
-      name: society.name,
-      admin: {
-        connect: {
-          id: society.userId,
-        }
-      }
+    if (!society.profilePicture) {
+      society.profilePicture = "null";

```

Comment:
does prisma let you just use `null`?

Reply:
I believe so, it's showing up as NULL on supabase. I could change it to smth safer though if needed(maybe setting a default value in the schema could be a better idea?).

Reply:
to clarify, do you mean `null` shows up as `NULL` or does `"null"` also translate to `NULL`?

Reply:
It looks like ```"null"``` is translating to ```NULL```. 

This is what it looks like on supabase:
<img width="231" alt="Screenshot 2024-12-04 at 9 00 40 PM" src="https://github.com/user-attachments/assets/3426a231-80b5-4ef2-97bc-cac0672cd37f">

Reply:
interesting, would you mind trying `null` on its own?

Reply:
```null``` translates to ```NULL``` if entered as an input on insomnia. If I remove the quotation marks in index.ts typescript gets mad.

Reply:
what happens if you open `backend/tsconfig.json` and set `strictNullChecks` to `false`?

Reply:
Oh nice, it's not angry anymore. I'll keep it as ```null```.